test/components/collections/list.js

```diff
@@ -57,20 +57,42 @@ const locationQueryParams = {
   search: {}
 };
 
+const urlHelper = {
```

Comment:
Why do we need to mock the urlHelper? What would be the problem if we imported the actual module?

Reply:
I thought it was the better way to integrate it with what was here since the Collection List component uses `urlHelper` as a prop.

Reply:
What would happen if we imported and used the real urlHelper instead of a mock?